lms/djangoapps/teams/views.py

```diff
@@ -315,6 +324,13 @@ def get(self, request):
                 status=status.HTTP_400_BAD_REQUEST
             )
 
+        if 'text_search' in request.QUERY_PARAMS and \
```

Comment:
We try to avoid \

I think you can do this as 
if ('text_search' .. and 
    'order_buy' ...):